src/layouts/sidebar-sidecar/sidebar-sidecar-layout.module.css

```diff
@@ -82,6 +81,18 @@ for further details.
   }
 }
 
+.sidebarContentWrapper {
+  position: relative;
+  padding: 24px;
+}
+
+.docsVersionSwitcherWrapper {
```

Comment:
Will +1 @BRKalow 's comment, realized [the earlier comment thread](https://github.com/hashicorp/dev-portal/pull/491#discussion_r885829405) wasn't totally accurate - it does seem the issues are happening because of a combination of the `70px` `padding-bottom` that Bryce mentioned, I think in addition to the issues with `var(--sticky-bars-height)` being inaccurate on smaller viewports due to text wrapping.

If we want to solve in this PR, I think a quick fix would be to set explicit heights on the top bars, to force sizes to reflect what we're setting in CSS (details in video on [this task](https://app.asana.com/0/1202114367927919/1202316169578550/f)).

However, I'm very okay with deferring getting the spacing right on mobile to a different PR - it feels tied up in a bunch of other tickets.

⛏️  To make sure this thread doesn't get lost, we could add a comment here in CSS in `sidebar-sidecar-layout.module.css` to note that mobile styles aren't quite done yet.